sources/packages/web/src/components/generic/BodyHeader.vue

```diff
@@ -19,12 +23,18 @@
 
 <script lang="ts">
 import { computed, defineComponent } from "vue";
+import DynamicHeader from "./DynamicHeader.vue";
 export default defineComponent({
+  components: { DynamicHeader },
   props: {
     title: {
       type: String,
       required: true,
     },
+    titleHeaderLevel: {
+      type: Number,
+      default: 1,
+    },
```

Comment:
Considering the primary usage of body header being in the body, can the default be H2?

@andrewsignori-aot @guru-aot @sh16011993 @lewischen-aot @katrina-aot-git ?

Reply:
Sounds good to me to have the default as 2.

Reply:
Yeah, if it is the first heading of a section but not the page it should probably be h2.

Reply:
Done. Thanks!